can/interface.py

```diff
@@ -27,21 +23,12 @@ def __new__(cls, other, channel=None, *args, **kwargs):
             can.rc['interface'] = kwargs['bustype']
```

Comment:
Yes - I agree in principal. A read only global config variable would be an improvement. I'll take a look and merge in the 'neovi' + io changes.